exercises/operations/operations1.cairo

```diff
@@ -18,11 +18,6 @@ fn poly(x: usize, y: usize) -> usize {
 #[test]
 fn test_poly() {
     let res = poly(5_usize, 3_usize);
-    assert(res == 126_usize, 'Error message');
-    assert(res < 300_usize, 'res < 300');
-    assert(res <= 300_usize, 'res <= 300');
-    assert(res > 20_usize, 'res > 2');
-    assert(res >= 2_usize, 'res >= 2');
-    assert(res != 27_usize, 'res != 27');
+    assert(res == 126_usize, 'Incorrect result');
```

Comment:
I don't really agree with this change, as the goal was to show an example of all comparison operators available

Reply:
Aah, sheesh. Thanks!